ddtrace/llmobs/_llmobs.py

```diff
@@ -75,6 +89,164 @@ def disable(cls) -> None:
         cls.enabled = False
         log.debug("%s disabled", cls.__name__)
 
+    def _start_span(
+        self,
+        operation_kind: str,
+        name: Optional[str] = None,
+        session_id: Optional[str] = None,
+        model_name: Optional[str] = None,
+        model_provider: Optional[str] = None,
+    ) -> Span:
+        if name is None:
+            name = operation_kind
+        span = self.tracer.trace(name, resource=operation_kind, span_type=SpanTypes.LLM)
+        span.set_tag_str(SPAN_KIND, operation_kind)
+        if session_id is not None:
+            span.set_tag_str(SESSION_ID, session_id)
+        if model_name is not None:
+            span.set_tag_str(MODEL_NAME, model_name)
+        if model_provider is not None:
+            span.set_tag_str(MODEL_PROVIDER, model_provider)
+        return span
+
+    @classmethod
```

Comment:
Should these methods also have docstrings? I know there are actual docs themselves in the works/that can be referred to, but wondering if a copypasta/short version here could be beneficial to user experience with the SDK as well.

Reply:
Yes they should, good catch!